File: testJob.php
#!/usr/bin/php
<?php
$payload="";
while($f = fgets(STDIN)) {
    $payload.=$f;
}

print "Got payload: $payload\n";
system("id");
for ($i=1; $i<=10; $i++)
{
    print "Doing stuph $i\n";
    sleep(1);
}
    fwrite(STDERR, "Diz is error\n\nfoobar\n");
print "Foobarstop\n";
die(123);

File: testJob.py
#!/usr/bin/python -u

import sys
import time
import subprocess
import fileinput

payload = ""
for line in fileinput.input():
	payload = payload + line

print "Got payload: "+payload

print "SimpleQueue testjob"
subprocess.call(["id"])
for i in range(0,10):
	print "Doing stuph "+str(i)
	time.sleep(1)


sys.stderr.write("Diz is error\n")

print "Stopping execution with errorcode 123\n"
sys.exit(123)
